docs/README.md

```diff
@@ -76,7 +76,7 @@ rustup target add wasm32-unknown-unknown
 In this tutorial you will:
 
 1. Bootstrap your development environment.
-2. Create, build, and deploy an index to an indexer service hooked up to Fuel's `beta-2` testnet.
+2. Create, build, and deploy an index to an indexer service hooked up to Fuel's `beta-3` testnet.
```

Comment:
Per the discussion in https://github.com/FuelLabs/fuel-indexer/issues/590, I don't think `beta-3` is ready yet?